libs/barista-components/checkbox/BUILD.bazel

```diff
@@ -1,7 +1,32 @@
-load("//tools/bazel_rules:index.bzl", "stylelint")
+load("@io_bazel_rules_sass//:defs.bzl", "sass_binary", "sass_library")
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("//tools/bazel_rules:index.bzl", "jest", "ng_module_view_engine", "stylelint")
 
 package(default_visibility = ["//visibility:public"])
 
+ng_module_view_engine(
+    name = "compile",
+    srcs = glob(
+        include = ["**/*.ts"],
+        exclude = [
+            "**/*.spec.ts",
+            "src/test-setup.ts",
+        ],
+    ),
+    angular_assets = [
+        ":styles",
+        ":src/checkbox.html",
+    ],
+    module_name = "@dynatrace/barista-components/checkbox",
+    tsconfig = "tsconfig_lib",
+    deps = [
+        "//libs/barista-components/icon:compile",
+        "@npm//@angular/core",
+        "@npm//@angular/common",
+        "@npm//@angular/cdk"
```

Comment:
please try to run first with `ng_module` and a `bazel build //libs/barista-components/checkbox:compile` as you miss some dependencies and this will provide you with the strict deps checks.

- rxjs
- @angular/forms
- form-field
- core